### PR TITLE
Add --skipcache option to id command

### PR DESCRIPTION
**Pull request by zanderz**
@maxtaco @patrick

### CODE REVIEW

File: go/client/cmd_id.go [hunk 1]
```diff
@@ -33,15 +34,17 @@ func (v *CmdID) ParseArgv(ctx *cli.Context) error {
 	}
 	v.trackStatement = ctx.Bool("track-statement")
 	v.useDelegateUI = ctx.Bool("ui")
+	v.skipCache = ctx.Bool("skip-cache")
```
Comment (patrickxb) [inline go/client/cmd_id.go]: Could we rename this `skip-proof-cache`?  There's another cache for the identify results and don't want callers to get confused about which cache they are skipping...

**Comment by patrickxb** [comment]
LGTM, just a rename request on the flag name.

File: go/client/cmd_id.go [hunk 2]
```diff
@@ -89,6 +92,11 @@ func NewCmdID(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
 				Usage:     "Use identify UI.",
 				HideUsage: !develUsage,
 			},
+			cli.BoolFlag{
+				Name:      "s, skip-proof-cache",
+				Usage:     "Skip cached proofs, force re-check",
+				HideUsage: !develUsage,
```
Comment (zanderz) [inline go/client/cmd_id.go]: hey I just realized, I copied this from another command - is this supposed to be only for devel?

Comment (patrickxb) [inline go/client/cmd_id.go]: I'm not sure.  Let's start with devel only for now, as that sounds like what the ticket is for.

**Comment by zanderz** [comment]
Made it user visible with:
commit 55ec37f5548cf5022f847ce78cf40dccf261575b
Author: Steve Sanders <steve@keyba.se>
Date:   Mon Mar 21 13:53:55 2016 -0700

    make skip-proof-cache option user visible

**Pull request merged**